news/editions/edition3.md

```diff
@@ -0,0 +1,67 @@
+
+## [Adopting the V2 K8S Provider As V1 Support Sunsets](https://blog.spinnaker.io/farewell-to-the-kubernetes-v1-provider-79d93861c6e4)
+Spinnaker 1.21, expected near the end of June, will be the final release that includes the Kubernetes V1 provider, as community best practices optimize for V2. Read more about generating V2-compliant, version-controlled Kubernetes manifests, and additional steps to migrate pipelines.
+
+## [PaCRD Deep Dive: Try Out Spinnaker Pipelines as CRDs](https://www.youtube.com/watch?v=HZtRCyGA7yM)
+Armory released a new "PaCRD" experience for experimental use this month. Watch to learn how to add a controller to your Spinnaker installation to manage application and pipeline objects as CRDs within your Kubernetes cluster. Store your pipeline definitions in the same format as other application resources, and provide your feedback.
+
+## [Spinnaker Summit CFP Now Open](https://linuxfoundation.smapply.io/prog/spinnaker_summit_2020/)
+With Spring upon us, the annual Spinnaker Summit is closer than you think. The CFP recently opened, and the application contains plenty of inspiration. Newly created tracks - Deep Dive, Hands-On, and Solutions & Case Studies - aim to create a high-quality program that will appeal to any CD stakeholder. Visit the #spinnaker-summit-2020 channel for brainstorming support, and count on help with presentation prep as needed. Submit your proposals today!
+
+## [Fix Applied for Clouddriver SQL Injection Bug Discovered by Autodesk](https://github.com/spinnaker/clouddriver/pull/4435)
+Engineers from Netflix and Armory acted quickly to include an important Clouddriver fix in the 1.17 release. The PEN testing team at longtime Spinnaker user company Autodesk discovered the SQL injection vulnerability and facilitated the quick fix. Community teamwork win!
+
+
+## User Stories
+## [Kubernetes-Native Spinnaker Pipelines with Istio at Descartes Labs](https://cd.foundation/blog/2020/02/24/descartes-labs-implementation-of-spinnaker-pipelines-the-end-of-waterfall/)
+Watch Louis Vernon's story of how Descartes Labs, an uber-cool geospatial analysis platform, evolved its waterfall deployments into modern Kubernetes pipelines, using GKE with Spinnaker, Istio and StackDriver.  Today, Istio routes updates between environments all running in the same cluster to deliver a stable SDLC.
+
+## [Multi-environment Microservice Delivery With Spinnaker at JPMC](https://www.infoq.com/presentations/spinnaker-jpmorgan-chase/)
+Hear Richard Francois, a VP at JP Morgan Chase, and Olga Kundzich explain how Spinnaker has helped the bank deploy software experiences to both public clouds and on-prem private clouds from one central locus of control and visibility. Monitoring system integration and no-code, zero-downtime deployments to Kubernetes, AWS, and the private cloud code-named "Gaia" have set this financial giant apart.
```

Comment:
For parallelism, it might be good to have a qualifier after Olga's name too.

Reply:
Yeah, I agree but didn't want to make it too long. Will add